README.md

```diff
@@ -201,6 +195,23 @@ PayPal authentication occurs in a popup window. However, this causes issues with
 
 PopupBridge solves this problem by allowing [`braintree-web`](https://github.com/braintree/braintree-web) to open the PayPal popup from a secure mini-browser.
 
+Using PayPal in a WebView
+-------------------------
+
+WebView-based checkout flows can accept PayPal with PopupBridge the Braintree JS SDK. For the authentication flow, PayPal requires a popup window—which can be simulated with PopupBridge.
```

Comment:
`WebView-based checkout flows can accept PayPal with PopupBridge and the Braintree JS SDK.` and I might just omit the second sentence.